### ISSUE TITLE
examples/sequence-to-one.lua -- Softmax layer has too many outputs

### ISSUE THREAD

**Issue by levinboimtomer** [Element-Research/rnn#139]
line 24,
  :add(nn.Linear(hiddenSize, **nIndex**))
should be 
  :add(nn.Linear(hiddenSize, **nClass**))
since there are only nClass classes to predicted.

However, 
once this change is made NLL err no longer exhibits a noticeable decrease in the first iterations.
Probably since the dataset is quite random.

**Comment by nicholas-leonard** [comment]
@levinboimtomer Good catch. Fixed in https://github.com/Element-Research/rnn/commit/71c901fff5a521bfa82f7e89b479d4d94ecdf7d3 . Thanks!
